group_vars/solr9cloud/staging.yml

```diff
@@ -5,129 +5,29 @@ install_zookeeper_from_source: true
 desired_ruby_version: "3.1.0"
 ruby_version_override: "ruby-3.1.0"
 solr_heap_setting: '20g'
+jardirectory: "/opt/solr/modules/analysis-extras/lib"
 # more granular option would be:
 # solr_java_memory: '-Xms16g -Xmx20g'
+# solr version
 solr_cloud_download_version: 9.2.0
+solr_cloud_version: "{{ solr_version }}"
+solr_version: "{{ solr_cloud_download_version }}"
 solr_log4j_path: '/solr/log4j2.xml'
-lib_zk4_host_name: lib-zk-staging4
-lib_zk5_host_name: lib-zk-staging5
-lib_zk6_host_name: lib-zk-staging6
-lib_zk4_host: '{{ vault_lib_zk_staging4_host }}'
-lib_zk5_host: '{{ vault_lib_zk_staging5_host }}'
-lib_zk6_host: '{{ vault_lib_zk_staging6_host }}'
 solr_znode: solr9
-solr_zookeeper_hosts_string_default: localhost:{{ solr_zookeeper_client_port }}
-solr_zookeeper_hosts_string_multi: '{{ lib_zk4_host_name }}:{{ solr_zookeeper_client_port }},{{ lib_zk5_host_name }}:{{ solr_zookeeper_client_port }},{{ lib_zk6_host_name }}:{{ solr_zookeeper_client_port }}'
-solr_zookeeper_hosts_string: '{{ solr_zookeeper_hosts_string_default if lib_zk4_host_name == "localhost" else solr_zookeeper_hosts_string_multi }}'
-zookeeper_env.{{ items }}:
-  - key: ZOO_LOG_DIR
-    value: "{{ zookeeper_log_dir }}"
-  - key: ZOO_LOG4J_PROP
-    value: "INFO,ROLLINGFILE"
-  - key: SERVER_JVMFLAGS
-    value: "-Xms2048m -Xmx2048m -verbose:gc -XX:+PrintGCDetails -Xlog:gc:$ZOO_LOG_DIR/zookeeper_gc.log"
-zookeeper_hosts_dict:
-  - host: '{{ lib_zk5_host_name }}'  # The leader in the Zookeeper ensemble
+solr_nodes:
+  - "lib-solr-staging1.princeton.edu"
+  - "lib-solr-staging2.princeton.edu"
+  - "lib-solr-staging3.princeton.edu"
+zookeeper_hosts:
+  - "lib-zk-staging4.princeton.edu:2181"
```

Comment:
Is the only difference between the `zookeeper_hosts` var and the `zk_nodes` var the addition of the port? Could we add the port in  the template that consumes the variable and avoid the duplication?

Reply:
we do not need any of these variables